src/store/transaction-details/transaction-details.thunks.js

```diff
@@ -47,7 +47,12 @@ function fetchTransaction(transactionIdOrHash) {
 
     return transactionPromise
       .then((res) => {
-        if (res.fromBJJ !== wallet.publicKeyBase64 && res.toBJJ !== wallet.publicKeyBase64) {
+        if (
+          res.fromBJJ !== wallet.publicKeyBase64 &&
+          res.toBJJ !== wallet.publicKeyBase64 &&
+          res.fromHezEthereumAddress !== wallet.hermezEthereumAddress &&
+          res.toHezEthereumAddress !== wallet.hermezEthereumAddress
+        ) {
```

Comment:
I think we should take this opportunity to improve the semantics of this conditional. The conditions above are hard to follow for newcomers.

Imagine we abstract these 4 conditions to variables:

```typescript
const A = res.fromBJJ !== wallet.publicKeyBase64;
const B = res.toBJJ !== wallet.publicKeyBase64;
const C = res.fromHezEthereumAddress !== wallet.hermezEthereumAddress;
const D = res.toHezEthereumAddress !== wallet.hermezEthereumAddress;
```

How would you name the variables `A`, `B`, `C`, and `D`?